File: gareth-api/src/main/java/org/craftsmenlabs/gareth/api/execution/ExecutionStatus.java
package org.craftsmenlabs.gareth.api.execution;

public enum ExecutionStatus {
    /**
     * return after successfully executing the baseline step
     */
    RUNNING,
    /**
     * Returned after the assume step was evaluated successfully
     */
    SUCCESS,
    /**
     * Returned after the assume step was evaluated with a failure
     */
    FAILURE
}

File: gareth-execution-ri/src/main/kotlin/org/craftsmenlabs/gareth/execution/dto/ExecutionStatus.kt
package org.craftsmenlabs.gareth.execution.dto

enum class ExecutionStatus {
    /**
     * return after successfully executing the baseline step
     */
    RUNNING,
    /**
     * Returned after the assume step was evaluated successfully
     */
    SUCCESS,
    /**
     *Returned after the assume step was evaluated with a failure
     */
    FAILURE
}